src/index.html

```diff
@@ -11,7 +11,248 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+      action="http://localhost:8080/"
+      method="post"
+      onsubmit="onSubmit()"
+    >
+
+      <fieldset class="box">
+        <legend>
+          Personal information
+        </legend>
+
+        <label
+          autocomplete="off"
+          for="surname"
+          class="form-field form"
+        >
+          Surname:
+
```

Comment:
```suggestion
          Surname:
```
Нужна ли тут эта пустая линия?

Reply:
Больше нет. Исправлено